app/renderer/components/Inspector/Commands.js

```diff
@@ -17,13 +17,27 @@ const Commands = (props) => {
     }
   };
 
+  const parseJsonString = (jsonString) => {
+    try {
+      return JSON.parse(jsonString);
+    } catch (e) {
+      notification.error({
```

Comment:
I assume it would be also helpful to show the error message itself

Reply:
Added the error message in the notification description